src/content/en/fundamentals/engage-and-retain/push-notifications/permissions-subscriptions.markdown

```diff
@@ -180,3 +183,34 @@ if ('showNotification' in ServiceWorkerRegistration.prototype) {
   });
 }
 {% endhighlight %}
+
+### Upgrading from GCM to the web push protocal {#upgrad}
```

Comment:
This section also feels super out of place here. You are talking about triggering a push message here, where as in the previous section you were referring to send the subscription to the developers server.

The above is the same regardless of GCM or Other.